### PR TITLE
update s3 methods to be consistent

### PR DESCRIPTION
**Pull request by bailliem**
update s3 methods to be consistent
* risktables
* get_quantile
* get_summary
* get_COX_HR

changed files so object class = x and ensure that ... was present in function call for S3 methods

### CODE REVIEW

**Comment by bailliem** [comment]
Close. Rename plot function to visr

**Comment by bailliem** [comment]
Wrong pr closed

File: R/get_COX_HR.R [hunk 1]
```diff
@@ -17,8 +17,8 @@ get_COX_HR <- function(x, ...){
 } 
 
 
-#' @param survfit_object An object of class \code{survfit}
-#' @param update_formula Template which specifies how to update the formula of the survfit_object \code{\link[stats]{update.formula}}
+#' @param x An object of class \code{survfit_visr}
```
Comment (SHAESEN2) [inline R/get_COX_HR.R:20]: This needs to be changed back to survfit.

**Review by SHAESEN2 — CHANGES_REQUESTED** [review]
Can you pull in develop and make sure all the test pass as well?

**Review by bailliem — COMMENTED** [review]
Removed visr from survfit object reference

**Pull request merged**